lib/compilers/rga.ts

```diff
@@ -236,6 +236,16 @@ Please supply an ASIC from the following options:`,
                         break;
                     }
 
+                    // Check if this line of assembly corresponds to a label. If so, emit the asm line
```

Comment:
Great stuff! If this gets too much more complex then maybe extracting some tests (a la the various and lengthy regression/approval tests in the `test/filters-cases`) or some other tests might be great. If only to try and lock in the good behaviour! Not too worried for now, but personally I've found (some of) those tests a real boon to development as I can iterate quicker.

Reply:
Sounds good, I should learn how the test suite works on the next PR!